### PR TITLE
Change societal amenities color

### LINKED ISSUE #1991: New landuse=farmland rendering color too close of amenity=hospital
Hello, there.

I noticed a change in the color used to render landuse=farmland; I wasn't able to find references to this change in the last issues or pull requests, but it now seems too close of the amenity=hospital rendering to me: https://www.openstreetmap.org/#map=15/48.2899/6.1141 In this case, I was working on the landuses on this area, and I first thought I mis-modelled the landuses and mixed them with amenity=hospital. Improving the landuse=farmland rendering color, which was somewhat pinkish to me, was a good idea, but now I would say it became confusing.

Regards.

Edit: I found the corresponding pull request, #1701

### PR DESCRIPTION
**Pull request by Adamant36**
This PR changes the color of societal amenities to #fffced from #fbecd7. As it was, it didn't fit well with the new tones of other colors, because it was to bright or some such. Whatever the case, the new color looks way better. So, this PR should totally be approved. Closes #1991
before 1
after 1
before 2
after 2
P.S. I noticed there's an issue with the commits and files changed for some reason. I'll be fixing it shortly when I have the time.

### CODE REVIEW

**Comment by Tomasz-W** [comment]
For all interested in more test renderings at different zoom levels, you can find them here: https://github.com/gravitystorm/openstreetmap-carto/issues/1991#issuecomment-423797757

**Comment by kocio-pl** [comment]
Problems:
- gray items changeset is not needed here, it's just one-liner
- lch value in this line has changed, so please fix it

The result is much better:

https://www.openstreetmap.org/#map=15/52.2983/20.8425

Before

grnwvwkt

After

_ud12izq

**Comment by kocio-pl** [comment]
I would however try with more yellow, because now it's hard to recognize it from bare ground, like [here](https://www.openstreetmap.org/#map=17/52.22527/21.00267):

Before 

8axda8q


After

ellswtlt

**Comment by Adamant36** [comment]
@kocio-pl, thanks for the heads up. I noticed that was an issue. I keep having branch bleed over. I'll fix it when I get some time.

**Comment by Tomasz-W** [comment]
> I would however try with more yellow

@Adamant36 Please make some test renderings with #FFFFE5

**Comment by Adamant36** [comment]
#FFFFE5 tests. I can do more if necessary. 
next to farmland before
z15 next to farmland before
next to farmland after
z15 next to farmland after
z14 before
z14 before
z14 after
z14 after
z16 before
z16 before
z16 after
z16 after
z17 before
z17 before
z17 after
z17 after

**Comment by Adamant36** [comment]
@kocio-pl, I think I fixed the issues with the code. I couldn't get rid of the original bad commit. Plus, it says I added the code for bicycle repair stations and a few other things unrelated to the PR because of how I did it, but at least it doesn't have any conflicts anymore. Also, good suggestion about adding more yellow. It looks better.

**Comment by kocio-pl** [comment]
BTW: how would the previous societal color work now? Maybe it would be enough to have new farmland color?

**Comment by Tomasz-W** [comment]
@kocio-pl I don't understand?

**Comment by kocio-pl** [comment]
The one we had till latest farmland change. @imagico swapped them and tuned it for new farmland, but I haven't tested if touching societal was really needed.

**Comment by kocio-pl** [comment]
Simple revert of this color would not work at all:

t8gftrlr

**Comment by kocio-pl** [comment]
However adding a bit of yellow (I have used 5% of old parking yellow - `mix(#ffffe5, #f7efb7, 95%)` ) works OK for me against both bare ground and farmland:

kiw6bzci

ant8byl2
ygnczbz3
e00mkzsm

**Comment by kocio-pl** [comment]
My color is effectively #fffee3, so it looks very similar and #FFFFE5 is OK for me too, however label text should be probably darker, like 80% instead of 70%:

u8jzza0h

70%
4iw909vj

80%
vatelih9

**Comment by kocio-pl** [comment]
#FFFFE5 with 90% darker text looks even better, I think:

ft7axwdf

**Comment by kocio-pl** [comment]
This contrast is too big for such popular landuse type like school or hospital (similar case to yellow parkings everywhere). It can work however for some less common areas - I guess culture and entertainment could be OK. Additional problem is that a lot of schools have pitch, which makes the contrast a bit painful. 

This background is clear enough for me (that's why I was trying to tune original proposition with yellow and it has really helped).

**Comment by Adamant36** [comment]
@turnsole80, when I test rendering the yellow color it was actually easier to spot some social amenities at higher zoom levels then it currently is. Plus, they weren't hard to distinguish from similar colored things at all, at any zoom level. So, its not necessarily the case that they will be harder to spot or anything just because the color is changed or is similar to other colors. Which it really isn't.

**Comment by turnsole80** [comment]
Fair enough. I can see that there is a difference between social amenities and farms, so it should be okay from my point of view.

**Comment by kocio-pl** [comment]
@Adamant36 Could you update the code according to https://github.com/gravitystorm/openstreetmap-carto/pull/3412#issuecomment-425695508?

**Comment by Adamant36** [comment]
@kocio-pl, sure. I'll do it when I get a chance. Sorry for the delay.

**Comment by matthijsmelissen** [comment]
Nice improvement, thanks @Adamant36 !

**Comment by Adamant36** [comment]
@kocio-pl, I updated it. Hopefully its correct now. 
davis

File: landcover.mss [hunk 1]
```diff
@@ -51,7 +51,7 @@
 @power: darken(@industrial, 5%);
 @power-line: darken(@industrial-line, 5%);
 @sand: #f5e9c6;
-@societal_amenities: #fbecd7;   // Lch(94,12,80)
+@societal_amenities: #ffffe5;   // Lch(94,12,80)
```
[the remark this answers is not in the document]
Comment (Adamant36) [reply landcover.mss]: @kocio-pl, my bad. I fixed it.

**Comment by kocio-pl** [comment]
Much better now, thanks a lot!

**Pull request merged**